actors/util/adt/map.go

```diff
@@ -38,6 +38,8 @@ func (m *Map) Root() cid.Cid {
 	return m.root
 }
 
+// TODO: Caching / commit
```

Comment:
Please remove this TODO for now, keep notes of things we might do later elsewhere.